### DISCUSSION TITLE
Migration from TreoPim to AtroPim?

### DISCUSSION THREAD

**Discussion by Vladzimir** [atrocore/atrocore#180] [Q&A] [answered]
Is there a step-by-step instruction for migrating from TreoPim to AtroPim?

**Comment by atrolex** [comment, marked as answer]
There are a lot of changes in AtroPIM in comparison to TreoPIM. Unfortunately, there is no migration guide. You should install AtroPIM and transfer all your data manually.

**Reply by Vladzimir** [reply]
Is it possible to export / import entities?

**Reply by atrolex** [reply]
Sorry, but not. We work on new modules for import and export now, but these will not be compatible with TreoPIM. As I said, you should transfer all the data manually.
